### PR TITLE
jin115.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/120322

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-30 в 16 27 36

Снимок экрана 2022-05-30 в 16 27 48

</details><br/>

### CODE REVIEW

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Maybe `||jin115.com^$removeparam=ref` as commented, but IDK where the parameter is added.

File: AnnoyancesFilter/sections/subscriptions.txt [hunk 1]
```diff
@@ -41,6 +41,8 @@ fila.com#$#.signup-modal { display: none!important; }
 fila.com#$#.ui-widget-overlay { display: none!important; }
 fila.com#$#body { position: static!important; }
 gatdaily.com###custom_html-22
+jin115.com##aside + aside.section-box > .plugin-free_area
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/subscriptions.txt:44]: This should be in Social filter.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/subscriptions.txt [hunk 2]
```diff
@@ -41,6 +41,8 @@ fila.com#$#.signup-modal { display: none!important; }
...
+jin115.com##.plugin-follow_buttons
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/subscriptions.txt:45]: @zloyden Should follow button now be in subscription? (I don't see recommendation).

Comment (zloyden) [reply AnnoyancesFilter/sections/subscriptions.txt:45]: The recommendation is `Follow`

**Review by zloyden — APPROVED** [review]

**Pull request merged**